amundsen_application/static/js/components/NavBar/styles.scss

```diff
@@ -3,54 +3,77 @@
 .nav-bar {
   height: 48px;
```

Comment:
I believe the new navbar height is now 60px. I've updated this in my PR, so It's up to you if you want to update here as well.

Reply:
You're right. Good catch, I'll update.